service/cachemdw/cache.go

```diff
@@ -0,0 +1,171 @@
+package cachemdw
```

Comment:
missing package level comment
what does `mdw` stand for?

Reply:
I added `doc.go` for package-level docs